src/components/ec-metroline-item/ec-metroline-item.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <div
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-metroline-item` : 'ec-metroline-item'"
+    class="ec-metroline-item"
+    :class="{
+      'ec-metroline-item--active': isActive || ( isCompleted && isLast),
+      'ec-metroline-item--is-last': isLast
+    }"
+  >
+    <div
+      data-test="ec-metroline-item__status"
+      class="ec-metroline-item__status"
+    >
+      <div
+        data-test="ec-metroline-item__status-disk"
+        class="ec-metroline-item__status-disk"
+      >
+        <span
+          v-if="!isCompleted"
+          data-test="ec-metroline-item__status-disk-enumeration"
+        >
+          {{ index }}
+        </span>
+
+        <ec-icon
+          v-else
+          data-test="ec-metroline-item__status-disk-completed"
+          name="simple-check"
+          :size="14"
+        />
+      </div>
+
+      <div
+        v-if="!isLast"
+        data-test="ec-metroline-item__status-bar"
+        class="ec-metroline-item__status-bar"
+      />
+    </div>
+
+    <div
+      data-test="ec-metroline-item__content"
+      class="ec-metroline-item__content"
+    >
+      <div
+        data-test="ec-metroline-item__header"
+        class="ec-metroline-item__header"
+      >
+        <div
+          data-test="ec-metroline-item__header-heading"
+          class="ec-metroline-item__header-heading"
+        >
+          <slot name="heading" />
+        </div>
+
+        <div
+          v-if="isCompleted"
+          data-test="ec-metroline-item__header-sub-heading"
+          class="ec-metroline-item__header-sub-heading"
+        >
+          <slot name="sub-heading" />
+        </div>
+
+        <div
+          v-if="isCompleted"
+          data-test="ec-metroline-item__header-cta"
+        >
+          <slot name="header-cta" />
+        </div>
+      </div>
+
+      <div
+        v-if="!isNext"
+        class="ec-metroline-item__main"
+        data-test="ec-metroline-item__main"
+      >
+        <slot name="main" />
+      </div>
+
+      <div
+        v-if="!isNext"
+        class="ec-metroline-item__footer"
+        data-test="ec-metroline-item__footer"
+      >
+        <div
+          class="ec-metroline-item__footer-cta"
+          data-test="ec-metroline-item__footer-cta"
+        >
+          <slot name="footer-cta" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import EcIcon from '../ec-icon';
+import * as STATUS from '../../enums/metroline-status';
+
+export default {
+  name: 'EcMetrolineItem',
+  components: { EcIcon },
+  props: {
+    index: {
+      type: String,
```

Comment:
I remember during the refinement we commented that we could define it as a string in case in the future, somewhere else, an enumeration with letters (A, B, C, etc) or similar is used

Reply:
It was mentioned during refinement that this can be a latin character as well.

Reply:
Not sure if index is the best name then. What about `badgeText` or `ordinalNumber`? ordinal number can be a string (it can be `1.` `2.` `1st`, `2nd`, but also `a)`, `b)`)

Reply:
badgeText is relevant